### ISSUE TITLE
ONNX to TensorRT, used torch.topk ERROR

### ISSUE THREAD

**Issue by DonggeunYu** [NVIDIA/TensorRT#987]
I used torch.topk and translate pytorch to onnx.
But Error about topk.
How i do fix it?

~~~
class TheModelClass(nn.Module):
    def __init__(self):
        super(TheModelClass, self).__init__()

    def forward(self, x):
        x = torch.topk(x, 1, dim=0)
        return x

the_model_class = TheModelClass()
the_model_class.eval().cuda()
call = torch.tensor([1., 2., 3., 4. ,5.], device="cuda")
torch.onnx.export(the_model_class,
                      call,
                      "test.onnx",
                      export_params=True,
                      verbose=True,
                      opset_version=11,
                      do_constant_folding=False)
~~~

ERROR:
~~~
[TensorRT] WARNING: onnx2trt_utils.cpp:220: Your ONNX model has been generated with INT64 weights, while TensorRT does not natively support INT64. Attempting to cast down to INT32.
In node -1 (importTopK): UNSUPPORTED_NODE: Assertion failed: inputs.at(1).is_weights()
~~~

**Comment by lihaoxiang1989** [comment]
I'm able to run your code with no error. PyTorch-v1.7.1

**Comment by DonggeunYu** [comment]
I want know your cuda, cudnn, TensorRT Version.

**Comment by lihaoxiang1989** [comment]
> I want to know your cuda, cudnn, TensorRT Version.

cuda 10.2, cudnn8.0.3, tensorrt7.2.1

**Comment by DonggeunYu** [comment]
Are you use Opset=9?

Opset9 is no error. but opset 10 over version is error.

**Comment by lihaoxiang1989** [comment]
> Are you use Opset=9?
> 
> Opset9 is no error. but opset 10 over version is error.

No, opset=11. I have the exact same code from you, what's your PyTorch version

**Comment by DonggeunYu** [comment]
I used clone https://github.com/NVIDIA/TensorRT from master branch. (1.8.1)
